template/nuxt.config.js

```diff
@@ -49,6 +49,13 @@ module.exports = {
       // { hid: 'canonical', rel: 'canonical', href: canonicalUrl }
     ]
   },
+  script: [
+    {
+      src: isProd
+        ? 'https://cdn-pci.optimizely.com/js/16943591923.js'
```

Comment:
for my education.. is there any reason that this file has such a random name?

Reply:
the number corresponds to our project ID number, that's randomly assigned to us